From 642b001130828ce77bfc40250a13e859c5714791 Mon Sep 17 00:00:00 2001
From: Adriana Reus <adriana.reus@nxp.com>
Date: Tue, 1 Aug 2017 10:28:46 +0300
Subject: [PATCH 2286/5242] MLK-16124 clk: imx: clean-up: Add defines for gate
 enable bits.

commit  0a112f245be974bd155a7c980a9ceb0236de0af7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Use defines for gate enable bits instead of raw values.

Signed-off-by: Adriana Reus <adriana.reus@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-gate-scu.c |   11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/drivers/clk/imx/clk-gate-scu.c b/drivers/clk/imx/clk-gate-scu.c
index a38fa45..06b1bb7 100644
--- a/drivers/clk/imx/clk-gate-scu.c
+++ b/drivers/clk/imx/clk-gate-scu.c
@@ -35,6 +35,9 @@
  * parent - fixed parent.  No clk_set_parent support
  */
 
+#define CLK_GATE_SCU_HW_SW_EN	(BIT(0) | BIT(1))
+#define CLK_GATE_SCU_SW_EN	BIT(1)
+
 struct clk_gate_scu {
 	struct clk_hw hw;
 	void __iomem	*reg;
@@ -80,9 +83,9 @@ static int clk_gate_scu_enable(struct clk_hw *hw)
 	if (gate->reg) {
 		reg = readl(gate->reg);
 		if (gate->hw_gate)
-			reg |= (0x3 << gate->bit_idx);
+			reg |= (CLK_GATE_SCU_HW_SW_EN << gate->bit_idx);
 		else
-			reg |= (0x2 << gate->bit_idx);
+			reg |= (CLK_GATE_SCU_SW_EN << gate->bit_idx);
 		writel(reg, gate->reg);
 	}
 
@@ -101,9 +104,9 @@ static void clk_gate_scu_disable(struct clk_hw *hw)
 	if (gate->reg) {
 		reg = readl(gate->reg);
 		if (gate->hw_gate)
-			reg &= ~(0x3 << gate->bit_idx);
+			reg &= ~(CLK_GATE_SCU_HW_SW_EN << gate->bit_idx);
 		else
-			reg &= ~(0x2 << gate->bit_idx);
+			reg &= ~(CLK_GATE_SCU_SW_EN << gate->bit_idx);
 		writel(reg, gate->reg);
 	}
 }
-- 
1.7.9.5

